src/contracts/BuildersManager.sol

```diff
@@ -237,12 +243,14 @@ contract BuildersManager is EIP712Upgradeable, Ownable2StepUpgradeable, IBuilder
   function _validateOptimismVoter(bytes32 _identityAttestation, address _claimer) internal returns (bool _verified) {
     if (eligibleVoter[_claimer]) revert AlreadyVerified();
     Attestation memory _attestation = EAS.getAttestation(_identityAttestation);
+    (,, string memory _voterType,,) = abi.decode(_attestation.data, (uint256, string, string, string, string));
+    bytes32 _voterTypeBytes = bytes32(bytes(_voterType));
 
     if (_attestation.uid == EMPTY_UID) return false;
-    if (!optimismFoundationAttester[_attestation.attester]) return false;
     if (_attestation.recipient != _claimer) return false;
+    if (_voterTypeBytes != _GUEST && _voterTypeBytes != _CITIZEN) return false;
 
-    _verified = true;
+    _verified = _attestation.schema == OP_SCHEMA_599;
```

Comment:
ideally this should not be hardcoded , a mapping for the schemas as well as a setter would be better for future maintainence